RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -38,3 +38,5 @@
                             ConvStep*
                             conversionStepTracks
                             )
+_iterTracking_Phase2PU140 = iterTracking.copyAndExclude([InitialStepPreSplitting])
+eras.trackingPhase2PU140.toReplaceWith(iterTracking, _iterTracking_Phase2PU140)
```

Comment:
The `recopixelvertexing` sequence is still added to the `globalreco_tracking` sequence in the customize function, e.g. https://github.com/ebrondol/cmssw/blob/ac4d4a97bbf859f02504a04f4d8d27a333a2e871/SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py#L255. As long as that is the case, dropping `InitialStepPreSplitting` is fine.

But doing the insertion of `recopixelvertexing` with eras we need to play some tricks (because of the way the jet-core pixel cluster splitting was implemented). I don't want to repeat the full discussion here (we can discuss next week if needed), so I'll just point to references
* comment in https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py#L176
* second paragraph of #13666 description and the links therein